### PR TITLE
Upgradeable Coordinator

### PR DESCRIPTION
**Pull request by vzotova**
**Type of PR:**
- [ ] Bugfix
- [X] Feature
- [ ] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
Makes `Coordinator` contract upgradeable

**Why it's needed:**
To upgrade `Coordinator` in the future instead of redeploying it

**Notes for reviewers:**
Based on #170

### CODE REVIEW

**Comment by derekpierre** [comment]
@vzotova FYI I'm looking into #169

**Review by KPrasch — APPROVED** [review]
🤠

**Review by KPrasch — APPROVED** [review]
Looking good 👍🏻 Perhaps you can also address #152 in this PR?  not a blocker.

**Review by theref — APPROVED** [review]

File: contracts/xchain/PolygonChild.sol [hunk 1]
```diff
@@ -27,6 +27,7 @@ contract PolygonChild is FxBaseChildTunnel, Ownable {
 
     function setChildApplication(address _childApplication) public onlyOwner {
         childApplication = _childApplication;
+        renounceOwnership();
```
[the remark this answers is not in the document]
Comment (derekpierre) [reply contracts/xchain/PolygonChild.sol:30]: So once child application is set, no one can ever set it again on this contract.

**Review by derekpierre — APPROVED** [review]
🎸

**Review by manumonti — APPROVED** [review]
LGTM!

**Pull request merged**